modules/services/vector_tile.js

```diff
@@ -104,7 +106,8 @@ function loadTile(source, tile) {
         .replace(/\{switch:([^}]+)\}/, function(s, r) {
             var subdomains = r.split(',');
             return subdomains[(tile.xyz[0] + tile.xyz[1]) % subdomains.length];
-        });
+        })
+        .replace('{bbox}', `${bbox.minX},${bbox.minY},${bbox.maxX},${bbox.maxY}`);
```

Comment:
is the intention of this PR to support WMS urls? (#8327)

* If yes, shouldn't we use [the same projection logic](https://github.com/openstreetmap/iD/blob/91ee41bcae182e60e8a7211b38da783fcf9015b1/modules/renderer/background_source.js#L141-L159) as the background renderer? 
* If no, is there any documentation for the type of URL that you're trying to support? Otherwise there is a risk that implementing support for `{{bbox}}` using this format will make it more complicated to support #8327 in the future